jobs/process_unassigned_chunks.go

```diff
@@ -0,0 +1,74 @@
+package jobs
+
+import (
+	"fmt"
+	"github.com/gobuffalo/pop"
+	"github.com/getsentry/raven-go"
+	"github.com/oysterprotocol/brokernode/models"
+	"strconv"
+)
+
+var BundleSize = 10
+
+type ProcessChunksFunc func(dataMaps []models.DataMap)
+
+type ChunkProcessor struct {
+	processChunks ProcessChunksFunc
+}
+
+func init() {
+	NewChunkProcessor(processChunks)
+}
+
+func NewChunkProcessor(processChunks ProcessChunksFunc) *ChunkProcessor {
+	return &ChunkProcessor{processChunks: processChunks}
+}
+
+func ProcessUnassignedChunks(processChunks ProcessChunksFunc) {
+
+	chunks, err := GetUnassignedChunks()
+
+	if err != nil {
+		raven.CaptureError(err, nil)
+	} else {
+		processChunks(chunks)
+	}
+}
+
+func GetUnassignedChunks() (dataMaps []models.DataMap, err error) {
+
+	tx, err := pop.Connect("test")
```

Comment:
This won't work in a non test environment. You might be able to access the `DB` ? (not sure where it was injected for the stuff I built). Otherwise, you will have to pass in the `ctx` object when you initialize the job.

Reply:
I meant to pull this out but forgot.